Frontend-Angular/data-gpt-frontend/src/app/app.component.spec.ts

```diff
@@ -0,0 +1,29 @@
+import { TestBed } from '@angular/core/testing';
+import { RouterTestingModule } from '@angular/router/testing';
+import { AppComponent } from './app.component';
+
+describe('AppComponent', () => {
+  beforeEach(() => TestBed.configureTestingModule({
+    imports: [RouterTestingModule],
+    declarations: [AppComponent]
+  }));
+
+  it('should create the app', () => {
+    const fixture = TestBed.createComponent(AppComponent);
+    const app = fixture.componentInstance;
+    expect(app).toBeTruthy();
+  });
+
+  it(`should have as title 'data-gpt-frontend'`, () => {
```

Comment:
Should it still be called data-gpt-frontend? Maybe make it more configurable

Reply:
This must be changed to develop instead of main asap